crates/rrg/src/args.rs

```diff
@@ -59,6 +59,16 @@ pub struct Args {
     pub command_verification_key: Option<ed25519_dalek::VerifyingKey>,
 }
 
+#[derive(Debug)]
+struct HexDecodeError();
```

Comment:
No need for `()`, you can just have `struct Foo;` (you will also need to change creation not to use parens).

Reply:
Done.